module/VuFind/src/VuFind/Db/Row/User.php

```diff
@@ -478,6 +479,8 @@ public function libraryCardsEnabled()
      *
      * @return \Laminas\Db\ResultSet\AbstractResultSet
      * @throws \VuFind\Exception\LibraryCard
+     *
+     * @deprecated Use UserCardServiceInterface::getLibraryCards()
```

Comment:
Because of incompatible return types, I didn't edit this method to be a wrapper around the new one; I just deprecated it as-is.